docs/internal/instrumentation-libraries.md

```diff
@@ -59,7 +59,7 @@ submit an issue to request its inclusion.
 | [CoreWCF](https://github.com/CoreWCF/CoreWCF)                                     |                                                                              |                                                                               |                                                                                                                                                        |
 | [RestSharp](https://restsharp.dev/)                                               |                                                                              |                                                                               | This library may be implicitly supported by instrumenting the underlying HttpClient or HttpWebRequest.                                                 |
 | [gRPC-dotnet](https://github.com/grpc/grpc-dotnet)                                |                          Partial[^grpc_client_only]                          |                                                                               | Client and service support should be added.                                                                                                            |
-| [GraphQL](https://www.nuget.org/packages/GraphQL/)                                |                     Partial (versions ≥2.3.0 & < 3.0.0)                      |                                                                               | The current instrumentation needs updates to match the semantic conventions.                                                                           |
+| [GraphQL](https://www.nuget.org/packages/GraphQL/)                                |                                     Yes                                      |                                                                               |                                                                                                                                                        |
```

Comment:
Specify version?  Specify .NET 5+?

Reply:
There's different doc for that https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2681/files#diff-77b3519370ff48f6caf7c316f0ece966f2f0871c40e9381d451d2c5b1c76b40fR130

But according to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2681#discussion_r1238544573, the information has to be adjusted.